schema/ont_papers.ttl

```diff
@@ -0,0 +1,114 @@
+@prefix : <https://w3id.org/ocs/ont/papers#> .
```

Comment:
You seem to have overwritten my changes – I have added language tags for comments and added a few new axioms. Please fix this – go to your git history and find my changes. Next time, please make sure you pull before making more changes – there may be more than one person working on your branch.